src/components/Persona/PersonaSignup.js

```diff
@@ -0,0 +1,272 @@
+import React, { useState } from 'react';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import cx from 'classnames';
+
+import {
+  Button,
+  Icon,
+  Trans,
+  useTessen,
+  useTranslation,
+} from '@newrelic/gatsby-theme-newrelic';
+import { createAccountRequest } from '@newrelic/gatsby-theme-newrelic/src/components/SignupModal/signup';
+import RecaptchaFooter from '@newrelic/gatsby-theme-newrelic/src/components/SignupModal/RecaptchaFooter';
+
+const MOBILE_BREAKPOINT = '600px';
+
+/**
+ * [VSU] This component allows users to sign up inline in a doc.
+ * It only renders if the current user is not logged in.
+ */
+export const PersonaSignup = ({ className }) => {
+  const [error, setError] = useState(null);
+  const [open, setOpen] = useState(true);
+  const tessen = useTessen();
+  const { t } = useTranslation();
+  const submitEvent = {
+    category: 'PersonaHomepageSignup',
+    eventName: 'attemptedSignup',
+  };
+
+  const onSubmit = async (e) => {
+    e.preventDefault();
+    const { email, name } = Object.fromEntries(new FormData(e.target));
+    const organizationId = await createAccountRequest(
+      { email, name },
+      tessen,
+      submitEvent
+    );
+
+    if (organizationId) {
+      const redirectUrl = `https://newrelic.com/thank-you?org=${organizationId}&standalone=true`;
+      window.location.assign(redirectUrl);
+    } else {
+      setError(true);
+    }
+  };
+
+  return (
+    <Wrapper open={open}>
+      <Form onSubmit={onSubmit} className={className}>
+        <CTAText>{t('personaSignup.terms')}</CTAText>
+        <Close name="x" onClick={() => setOpen(false)} />
```

Comment:
this should probably be a `button` with `aria-label="Close"` for assistive technologies